pandas/core/frame.py

```diff
@@ -5982,7 +5982,8 @@ def infer(x):
     # ----------------------------------------------------------------------
     # Merging / joining methods
 
-    def append(self, other, ignore_index=False, verify_integrity=False):
+    def append(self, other, ignore_index=False,
+               verify_integrity=False, sort=False):
```

Comment:
sort before verify_integrity

Reply:
@jreback why do you want sort before verify_integrity?